DEFAULT_CONFIG.json5

```diff
@@ -412,12 +412,26 @@
             drop: {
               /// The maximum time in microseconds to wait for an available batch before dropping a droppable message if still no batch is available.
               wait_before_drop: 1000,
+              /// The maximum deadline limit for multi-fragment messages.
+              /// When sending multi-fragment message, for each consecutive fragment the deadline from
+              /// the start point of message transmission is calculated as 
+              /// wait_before_drop * 2 ^ min(N, ttl) where N is a fragment number.
+              /// Thus, this parameter allows setting a limitation for maximum time for multi-fragment
+              /// message transmission as wait_before_drop * 2 ^ ttl
+              ttl: 8,
```

Comment:
I'd expose the configuration option differently without the need of leaking internal details about exponential backoff. 
What about calling it something like `max_wait_before_drop_fragments`?

Reply:
So, you suggest just representing the max limitation in milliseconds, completely hiding the inner calculations? Yes, it makes sense!